fuzz/src/fuzz.java

```diff
@@ -0,0 +1,21 @@
+import com.github.bhlangonijr.chesslib.Board;
```

Comment:
Can we rename this file to reflect what's being fuzzed.

Reply:
I've renamed it